backend/build.gradle

```diff
@@ -35,12 +35,14 @@ dependencies {
 	implementation 'org.json:json:20201115'
 	implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-csv'
 
-//	Okta dependencies
+	//	Okta dependencies
 	implementation 'com.okta.spring:okta-spring-boot-starter:1.4.0'
 
-// App insights instrumentation
+	// App insights instrumentation
 	implementation 'com.microsoft.azure:applicationinsights-core:2.6.2'
 
+	// Cron dependencies
+	implementation "org.springframework.boot:spring-boot-starter-batch"
```

Comment:
I don't think you actually need spring batch for this—the scheduling stuff is all in core Spring, AFAIK.

Reply:
You are correct. Documentation and examples all indicated it was needed. e.g. https://spring.io/projects/spring-batch 

I removed it and it seems to work. Removing it from the PR.